models/brac/useviews/useview_health_forum.sql

```diff
@@ -1,14 +1,16 @@
 {{
     config(
         materialized = 'incremental',
-        unique_key="uuid",
+        unique_key="xmlforms_uuid",
```

Comment:
This will fail since we keep track of doc revisions as well.

With native materialized views, I understand the need of a unique index. Are they required for incremental tables in DBT or would a normal index suffice?

Reply:
Noted.